src/devices/CharacterLcd/samples/PCF8574T Sample/PCF8574T Test/Program.cs

```diff
@@ -0,0 +1,357 @@
+﻿using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+using System.Timers;
+using Iot.Device.Mcp23xxx;
+using Iot.Device.CharacterLcd;
+
+
+
+namespace test
+{
+    class Program
+    {
+        public class ExtendedSample

```

Comment:
is this sample mostly the same as original? (except for using extender) If so, it might make sense to `#ifdef` those few lines of code and only create a separate csproj in the same folder which defines the constant or just extract one method which differs to separate file and include file which differs in respective project

Reply:
Instead of creating new project, why not just add a new file to the samples project that already exists, named Pcf8574tSample.cs and have this code there? To me there is very low value on having two different sample projects.

Reply:
hi guys

I created a new solution as it was the only efficient way to test the numerous itinerations of changes i had to do to get this working. but esentially yes this is the extended sample with correct pin assignments for the PCF IC.

sure i can put just a sample file there, do i just make the changes and re push to my fork? or does a new PR need to be made too? 

@krwq I'm by no means a developer this is a little above my head i'd probably need help here if you wanted to go this route.

the only other suggestion i can make is omit the sample and just have the correct definition in the readme? essentially the code is all the same, you just need to change the setup for the lcd really.

re: spaces in file names and extra crlf thats easy enough fixed

Reply:
> sure i can put just a sample file there, do i just make the changes and re push to my fork? or does a new PR need to be made too?

Correct, all you have to o is to make the changes locally, commit them to your local branch, and then push to your fork. This PR will be auto updated with those changes so that we can take another look 😄. Feel free to ask any questions about the process, we are happy to help.

Reply:
Hi @joperezr 

OK, I've changed the sample and it should be part of the current sample project; its named Pcf8574tSample.cs.

The extra crlf's should be all fixed too.

let me know if you think it needs anything else or any suggestions on improvements.